neo-proxy.js

```diff
@@ -95,3 +106,12 @@ function neoProxy(options) {
 
   return options
 }
+
+function rewriteUrlsFunction (host, localHost, port) {
+  var links = document.getElementsByTagName('a');
+  for (var i = links.length - 1; i >= 0; i--) {
+    var link = links[i],
+      href = link.getAttribute('href');
```

Comment:
one var per line is a good idea (explanation: http://benalman.com/news/2012/05/multiple-var-statements-javascript/ )